providers/okta/application_test.go

```diff
@@ -0,0 +1,33 @@
+package okta
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/riotgames/key-conjurer/api/cloud"
+	"github.com/riotgames/key-conjurer/api/core"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// This is an integration test which performs real authorization actions against a real endpoint.
+func Test_ApplicationSAMLSource_GetAssertion(t *testing.T) {
+	if _, ok := os.LookupEnv("CI"); ok {
+		t.Skipf("Not running %s in CI", t.Name())
+	}
+
+	ctx := context.Background()
+	source := ApplicationSAMLSource(os.Getenv("OKTA_APPLICATION_URL"))
+	assertionBytes, err := source.GetAssertion(ctx, os.Getenv("USER"), os.Getenv("PASS"))
```

Comment:
Did you know that OSX will not let you override the env var `USERNAME`? It caused a lot of headaches when going back between Ubuntu & OSX